test/google_scraping_web/controllers/keyword_controller_test.exs

```diff
@@ -85,4 +85,31 @@ defmodule GoogleScrapingWeb.KeywordControllerTest do
                "One or more keywords are invalid! Allowed keyword length is 1-100"
     end
   end
+
+  describe "GET show/2" do
+    test "when given auth user and keyword, renders keywords details page", %{conn: conn} do
+      user = insert(:user)
+      keyword = insert(:keyword, user_id: user.id, name: "test keyword")
+      conn = conn |> log_in_user(user) |> get(Routes.keyword_path(conn, :show, keyword))
```

Comment:
Another minor comment, but I think having these on separate lines would be easier to read and consistent with the rest of the tests in the file

```suggestion
      conn = 
         conn 
         |> log_in_user(user) 
         |> get(Routes.keyword_path(conn, :show, keyword))
```